kibble/api/films.go

```diff
@@ -252,27 +265,19 @@ type filmV2 struct {
 	Studio []struct {
 		Name string `json:"name"`
 	} `json:"studio"`
-	Overview    string   `json:"overview"`
-	Tagline     string   `json:"tagline"`
-	ReleaseDate string   `json:"release_date,omitempty"`
-	Runtime     float64  `json:"runtime"`
-	Countries   []string `json:"countries"`
-	Languages   []string `json:"languages"`
-	Genres      []string `json:"genres"`
-	Tags        []string `json:"tags"`
-	Title       string   `json:"title"`
-	Slug        string   `json:"slug"`
-	FilmID      int      `json:"film_id"`
-	ID          int      `json:"id"`
-	ImageUrls   struct {
-		Portrait       string `json:"portrait"`
-		Landscape      string `json:"landscape"`
-		Header         string `json:"header"`
-		Carousel       string `json:"carousel"`
-		Bg             string `json:"bg"`
-		Classification string `json:"classification"`
-		Seo            string `json:"seo"`
-	} `json:"image_urls"`
```

Comment:
Just confirming my understanding: there's no API change going from `struct` to `map[string]string` yeah? I.e. the `json` decriptors are smart enough to handle it.

A quick GitHub search suggests nothing uses this so fine to change types.

---

Out of interest (ignoring other differences in map and naming), I notice [the docs for show multiple](https://apidocs.shift72.com/services/meta/#film-meta-show-multiple) doesn't mention SEO image:

![image](https://user-images.githubusercontent.com/2253814/158901909-c3b47283-3e4e-43e4-a3ec-5ef671735e4b.png)

... while [show single](https://apidocs.shift72.com/services/meta/#film-meta) does:

![image](https://user-images.githubusercontent.com/2253814/158901952-ab454839-60d4-419a-baa3-c9422fb01130.png)